src/v2/providers/https.ts

```diff
@@ -40,7 +40,7 @@ export interface HttpsOptions extends Omit<options.GlobalOptions, 'region'> {
     | options.SupportedRegion
     | string
     | Array<options.SupportedRegion | string>;
-  cors?: string | boolean | RegExp | (string | RegExp)[];
+  cors?: string | boolean | RegExp | Array<string | RegExp>;
```

Comment:
Just curious, why do we prefer this syntax here?

When I touched this area, the underlying origin property of the cors library uses the bracket notation

Reply:
Commented below, `npm fix:lint` did it for us.